README.md

```diff
@@ -354,6 +365,10 @@ localStorage.setItem(
   "LocalSettings:Sales:CallingExtensions",
   '{"name": "Demo widget", "isReady": true}'
 );
+localStorage.setItem(
+  "LocalSettings:Calling:CallingExtensions",
+  '{"name": "Demo widget", "isReady": true}'
+);
```

Comment:
Might just want to add the same comments here as well:
`// Add the following localstorage override for the newer floating widget`
and 
`// Add the following localstorage override for the non movable demo widget`